unconference/LMIC-know-how.md

```diff
@@ -0,0 +1,200 @@
+    * The grant does not fit the remit of the call for proposals
+    * Writing and grammar skills
+    * Persons/Collaborators do not have the skills
+    * Unrealistic timeline or budget
+    * Application does not show experience with grant writing
+* What this material will NOT teach
+    * Comprehensive overview of grant writing (there are other grant writing workshops, and this won’t cover discipline-specific considerations)
+    * Complete information about program/project management and leadership (community management, managing personnel), and planning for organizational sustainability
+
+*Module 1 learning objectives:* 
+At the end of this module, learners should be able to:
+* Understand the scope of this curriculum
+* Understand how to use and reuse these materials 
+* Recognize the target the audience of this content
+* Identify common issues while seeking funding that this curriculum will address
+
+
+### Module 2: Preparing to apply for grants
+
+* What is a grant and what are the types of grants
+    * Competitive requests for applications (RFAs)
+    * Unsolicited submissions and pitching ideas
+* Meeting and getting to know funders (building trust, learning about program priorities)
+* Building a professional brand
+    * Website for lab, organization, or individuals
+    * Basic information about goals
+* Reading the RFA
+    * How the grant will be evaluated
+    * Frame your proposal around the evaluation rubrics
+* What do funders look for in candidates 
+* Organizational requirements
+    * Connecting with your institution and learning about procedures
+    * Finding or learning if you need to create an organization to support you (fiscal sponsorship)
+* Finding supporters that will give you help/feedback for your proposal
+* Templates
+    * Collaborator recommendation letters 
+    * CVs
+
+*Module 2 learning objectives:* 
+At the end of this module, learners should be able to:
+* Identify grant opportunities for which they are eligible
+* Identify key collaborators or stakeholders that need to be involved
+* Understand the basic requirements you need to meet for a grant application
+* Decypher the RFA specifics and translate them into actionable application tasks
+
+
+### Module 3: Preparing a proposal
+
+* Preparing a proposal
+    * Sections of a proposal
+    * Research narrative
+    * Diversity, equity, and inclusion (DEI) statement
+* Deliverables and outcomes
+* Letters of recommendation/support
+    * Formatting
+    * Aims/objectives/goals 
+    * Questions/hypotheses
+    * Methods
+    * Deliverables and success metrics
+    * Citation/references
+    * Supporting materials 
+    * Lexicon (how to handle acronyms)
+* Timelines
+* What makes a good proposal
+    * Consistency
+    * Concise but clear (not too specific or too general)
+* Resources at your affiliated institution or organization 
+* Institutional approval
+* Listing your collaborators
+* Steps in the application process
```

Comment:
these steps are again very funder (and opportunity) specific - e.g., most NSF calls don't ask for LOIs or have interviews. Some big projects have a site-visit, which is kind of like an oral presentation of the proposal and can include answers to some reviewer questions.